### ISSUE TITLE
Cycle toggle - Dynamic aria-selected doesn't work when value is last span

### ISSUE THREAD

**Issue by GalinhaLX** [LeaVerou/nudeforms#5]
Hi

Cycle toggle doesn't work when selected span is after the value we want to affect

in your page: [here](https://projects.verou.me/nudeforms/cycle-toggle/)

if you select  😀 (last span), you'll not be able to apply to 🙂

<img width="964" alt="image" src="https://user-images.githubusercontent.com/89095737/182414806-b13460d2-d5f3-4d37-b186-bfb1501bc44a.png">

**Comment by LeaVerou** [comment]
Yeah, you need to *remove* the existing `aria-selected` attribute first.
